cli/command/registry_test.go

```diff
@@ -77,3 +138,62 @@ func TestElectAuthServer(t *testing.T) {
 		}
 	}
 }
+
+func TestGetDefaultAuthConfig(t *testing.T) {
+	testCases := []struct {
+		checkCredStore     bool
+		expectErr          bool
```

Comment:
Instead of a boolean for the error could you use `expectedErr string`, and check that the error message contains that string.